### PR TITLE
Wagtail 3.0 compatibility

### PR DESCRIPTION
**Pull request by nickmoreton**
### Add support for Wagtail 3.0

This PR does mean the package now requires Wagtail 2.15+
I noticed some issues with the latest Jinja2 version so have pinned that and markupsafe

```
jinja2>=2.10,<3.0,
markupsafe==2.0.1,
```

Tasks

- [x] Update docs
- [x] Update Change Log
- [x] Change import paths
- [x] Update tests
- [x] Update tox config

### CODE REVIEW

File: tests/app/models.py [hunk 1]
```diff
@@ -1,7 +1,15 @@
 from django.db import models
-from wagtail.admin.edit_handlers import FieldPanel
+from wagtail import VERSION as WAGTAIL_VERSION
+
+if WAGTAIL_VERSION >= (3, 0):
+    from wagtail.admin.panels import FieldPanel
+    from wagtail.models import Page
+else:
+    from wagtail.admin.edit_handlers import FieldPanel
+    from wagtail.images.edit_handlers import ImageChooserPanel
```
Comment (BrianXu20) [inline tests/app/models.py:9]: I found it it's cleaner if keep `ImageChooserPanel('photo')` and `from wagtail.admin.panels import FieldPanel as ImageChooserPanel`. This is totally optional

**Review by BrianXu20 — APPROVED** [review]
LGTM! Good job.

**Comment by nickmoreton** [comment]
Hi @seb-b It would be great if you could review this for us please and get anew release out. Thank you.

File: setup.py [hunk 2]
```diff
@@ -22,7 +22,9 @@
     url='https://github.com/takeflight/wagtail-schema.org',
 
     install_requires=[
-        'wagtail>=2.0',
+        'wagtail>=2.15,<4.0',
+        'jinja2>=2.10,<3.0',
+        'markupsafe==2.0.1',
```
Comment (seb-b) [inline setup.py:27]: the jinja2 and therefore the markupsafe reqs should be set via the wagtail deps, I'm a bit concerned that there could be conflict resolution problems if these are pinned in a third party package. What problem does pinning these solve?

Comment (nickmoreton) [reply setup.py]: Ah thats a good point. I added them there when initially running the tests as they required the packages and should have included them as testing dependencies.

Hope this is a better solution. Thanks

**Comment by seb-b** [comment]
Thanks @nickmoreton! Just released version 3.0.0 with this pr in it

**Pull request merged**